<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>first_face</class>
 <widget class="QMainWindow" name="first_face">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/first_face/res/main_icon.png</normaloff>:/first_face/res/main_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="save"/>
   <addaction name="load"/>
   <addaction name="run"/>
   <addaction name="stop"/>
  </widget>
  <widget class="QDockWidget" name="online_logger">
   <property name="toolTip">
    <string>Online logger.</string>
   </property>
   <property name="statusTip">
    <string>Online logger.</string>
   </property>
   <property name="whatsThis">
    <string>Online logger.</string>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Online logger.</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTextEdit" name="text_shower">
       <property name="documentTitle">
        <string/>
       </property>
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="values_setter">
   <property name="toolTip">
    <string>Values' setter.</string>
   </property>
   <property name="statusTip">
    <string>Values' setter.</string>
   </property>
   <property name="whatsThis">
    <string>Values' setter.</string>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Values' setter</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="project_menu">
    <property name="toolTip">
     <string>The project's menu.</string>
    </property>
    <property name="statusTip">
     <string>The project's menu.</string>
    </property>
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="load"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="toolTip">
     <string>The help's menu.</string>
    </property>
    <property name="statusTip">
     <string>The help's menu.</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="about"/>
    <addaction name="license"/>
    <addaction name="guide"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="show_values_setter"/>
    <addaction name="show_online_logger"/>
   </widget>
   <widget class="QMenu" name="menu_plugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <widget class="QMenu" name="menu_open_plugin">
     <property name="title">
      <string>Open</string>
     </property>
    </widget>
    <addaction name="action_set_dir_search"/>
    <addaction name="separator"/>
    <addaction name="menu_open_plugin"/>
   </widget>
   <addaction name="project_menu"/>
   <addaction name="menu_view"/>
   <addaction name="menu_plugins"/>
   <addaction name="help_menu"/>
  </widget>
  <action name="save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/first_face/res/save.png</normaloff>:/first_face/res/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current project.</string>
   </property>
   <property name="statusTip">
    <string>Save the current project.</string>
   </property>
  </action>
  <action name="save_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save the current project as.</string>
   </property>
   <property name="statusTip">
    <string>Save the current project as.</string>
   </property>
  </action>
  <action name="load">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/first_face/res/load.png</normaloff>:/first_face/res/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a project.</string>
   </property>
   <property name="statusTip">
    <string>Load a project.</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>An info about the programm.</string>
   </property>
   <property name="statusTip">
    <string>An info about the programm.</string>
   </property>
  </action>
  <action name="license">
   <property name="text">
    <string>License</string>
   </property>
   <property name="statusTip">
    <string>This text of this project's license.</string>
   </property>
   <property name="whatsThis">
    <string>This text of this project's license.</string>
   </property>
  </action>
  <action name="guide">
   <property name="text">
    <string>Guide</string>
   </property>
   <property name="toolTip">
    <string>This programm's guide.</string>
   </property>
   <property name="statusTip">
    <string>This programm's guide.</string>
   </property>
  </action>
  <action name="run">
   <property name="icon">
    <iconset>
     <normaloff>res/run.png</normaloff>res/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Make the project running.</string>
   </property>
   <property name="statusTip">
    <string>Make the project running.</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close this program.</string>
   </property>
   <property name="statusTip">
    <string>Close this program.</string>
   </property>
  </action>
  <action name="show_values_setter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show values' setter</string>
   </property>
   <property name="toolTip">
    <string>Show values' setter.</string>
   </property>
   <property name="statusTip">
    <string>Show values' setter.</string>
   </property>
   <property name="whatsThis">
    <string>Show values' setter.</string>
   </property>
  </action>
  <action name="show_online_logger">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show online logger</string>
   </property>
  </action>
  <action name="stop">
   <property name="icon">
    <iconset>
     <normaloff>res/stop.png</normaloff>res/stop.png</iconset>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
   <property name="toolTip">
    <string>Stop calculating.</string>
   </property>
  </action>
  <action name="action_set_dir_search">
   <property name="text">
    <string>Set dir search</string>
   </property>
   <property name="toolTip">
    <string>Set initial for for searching any availaible model. </string>
   </property>
   <property name="statusTip">
    <string>Set initial for for searching any availaible model. </string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>show_values_setter</sender>
   <signal>triggered(bool)</signal>
   <receiver>values_setter</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>values_setter</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>show_values_setter</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>show_online_logger</sender>
   <signal>triggered(bool)</signal>
   <receiver>online_logger</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>online_logger</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>show_online_logger</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
